src/Entity/EntityContentTrait.php

```diff
@@ -55,6 +57,7 @@
      * @var string|null
      *
      * @ORM\Column(length=255, nullable=true)
+     * @Assert\Length(max=255)
```

Comment:
Please add an empty line between mapping and assertions.

Reply:
done